drivers/ethernet/eth_cyclonev.c

```diff
@@ -1145,13 +1145,13 @@ const struct ethernet_api eth_cyclonev_api = {.iface_api.init = eth_cyclonev_ifa
 
 #define CYCLONEV_ETH_INIT(inst) \
  \
-	static struct eth_cyclonev_priv eth_cyclonev_##inst##_data; \
+	static struct eth_cyclonev_priv eth_cyclonev_##inst##_data;	\
 	static void eth_cyclonev_##inst##_irq_config(void); \
  \
 	static const struct eth_cyclonev_config eth_cyclonev_##inst##_cfg = { \
 			.base = (uint8_t *)(DT_INST_REG_ADDR(inst)), \
 			.size = DT_INST_REG_SIZE(inst), \
-			.inst_num = inst, \
+			.emac_index = DT_INST_PROP(inst, emac_index), \
```

Comment:
These changes aren't related to the commit message `Change the bit label to include the BIT() macro to tidy up the code.`

Reply:
Fixed in [c38c85f](https://github.com/zephyrproject-rtos/zephyr/pull/53693/commits/c38c85f8300ee6eb263cf8d2a80d886a0d58f4ce) Changes reflected in [4f2ab80](https://github.com/zephyrproject-rtos/zephyr/pull/53693/commits/4f2ab80b3d3d648acb02d5a2af3bbc876d00b964)